cogs/queue_writer.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+async def update_queue_file(queue):
```

Comment:
This technically blocks the executor and should really spawn a new thread to perform this, but I don't know if the wavelink queue object is thread safe. Should be fine anyways.